docs/machine-learning/resources/transforms.md

```diff
@@ -0,0 +1,131 @@
+
+Click to jump immediately to a certain section of transforms:
+* [Categorical](#categorical)
+* [Combiners and segregators](#combiners-and-segregators)
+* [Feature selection](#feature-selection)
+* [Featurizers](#featurizers)
+* [Label parsing](#label-parsing)
+* [Missing Values](#missing-values)
+* [Normalization](#normalization)
+* [Row filters](#row-filters)
+* [Schema](#schema)
+* [Text processing and featurization](#text-processing-and-featurization)
+* [Miscellaneous](#miscellaneous)
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all data transforms are currently supported. To submit a request for a certain transform, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
+
+## Categorical
+
+| Transform | Definition |
+| --- | --- |
+| CategoricalHashOneHotVectorizer | Encodes the categorical variable with hash-based encoding. |
+| CategoricalOneHotVectorizer | Encodes the categorical variable with one-hot encoding based on term dictionary. |
+
+## Combiners and segregators
+
+| Transform | Definition |
+| --- | --- |
+| CombinerByContiguousGroupId | Groups values of a scalar column into a vector, by a contiguous group ID. |
+| FeatureCombiner | Combines all the features into one feature column. |
+| ManyHeterogeneousModelCombiner | Combines a sequence of TransformModels and a PredictorModel into a single PredictorModel. |
+| ModelCombiner	| Combines a sequence of TransformModels into a single model. |
+| Segregator | Un-groups vector columns into sequences of rows, inverse of Group transform. |
+| TwoHeterogeneousModelCombiner	| Combines a TransformModel and a PredictorModel into a single PredictorModel. |
+
+## Feature selection
+
+| Transform | Definition |
+| --- | --- |
+| FeatureSelectorByCount | Selects the slots for which the count of non-default values is greater than or equal to a threshold. |
+| FeatureSelectorByMutualInformation | Selects the top k slots across all specified columns ordered by their mutual information with the label column. |
+
+## Featurizers
+
+| Transform | Definition |
+| --- | --- |
+| HashConverter	| Converts column values into hashes. This transform accepts both numeric and text inputs, both single and vector-valued columns. This is a part of the Dracula transform. |
+| TreeLeafFeaturizer | Trains a tree ensemble, or loads it from a file, then maps a numeric feature vector to three outputs: 1. A vector containing the individual tree outputs of the tree ensemble. 2. A vector indicating the leaves that the feature vector falls on in the tree ensemble. 3. A vector indicating the paths that the feature vector falls on in the tree ensemble. If a both a model file and a trainer are specified - will use the model file. If neither are specified, will train a default FastTree model. This can handle key labels by training a regression model towards their optionally permuted indices. |
```

Comment:
If a both a model file --> If both a model file.
there should be a subject that corresponds to the verbs "will use" and "will train"